Dockerfile.build

```diff
@@ -0,0 +1,8 @@
+FROM openshift/origin-release:golang-1.7
+MAINTAINER Federico Simoncelli <fsimonce@redhat.com>
+
+COPY .  /go/src/github.com/openshift/image-inspector
+
+RUN GOBIN=/usr/bin \
+    CGO_ENABLED=0 \
+    go install -a -installsuffix cgo /go/src/github.com/openshift/image-inspector/cmd/image-inspector.go
```

Comment:
@mfojtik containers/image relies on containers/storage which also requires cgo (e.g.: https://github.com/containers/storage/blob/master/pkg/devicemapper/devmapper_wrapper.go#L5)

although this PR isn't the place to account for cgo requirements